CalibMuon/DTCalibration/python/Workflow/DTWorkflow.py

```diff
@@ -254,12 +260,12 @@ def addPoolDBESSource( process,
 
         calibDB = cms.ESSource("PoolDBESSource",
                                CondDB,
-                               timetype = cms.string('runnumber'),
+                               #timetype = cms.string('runnumber'),
```

Comment:
just to mention that this will conflict with https://github.com/cms-sw/cmssw/pull/47851

Reply:
Hmm, actually the opposite. The line is commented out not to conflict with that change.

Reply:
Ah, I see what you mean. This change is already made in that PR. That's fine. When there is time to merge, I will resolve the conflicts.